packages/react-vanilla-components/src/components/FileUpload.tsx

```diff
@@ -52,33 +67,52 @@ const FileUpload = (props: PROPS) => {
     return localFiles;
   }, []);
 
-  const handleNewFileUpload = useCallback((e: { target: { files: FileList | null; }; }) => {
-    const { files: newFiles } = e.target;
-    if (newFiles?.length) {
-      const updatedFiles = files.concat(fileListToArray(newFiles));
+  const handleDragOver = (event: React.DragEvent<HTMLDivElement>) => {
```

Comment:
I don't think this function is required.

Reply:
This is required. If it is removed the file gets open in other tab.